examples/energy_mng/README.md

```diff
@@ -0,0 +1,10 @@
+See the [MCU backend Readme](../mcu-board-support) to see how to run the example on a smaller device like the Raspberry Pi Pico.
+
+The example can run with the mcu simulator with the following command
+
+```cargo run -p carousel --no-default-features --features=simulator --release```
```

Comment:
wrong name there...  (carousel)


But btw we want to rename it to `energy-monitor`

Reply:
Yes I want to rename it after this PR in a separate PR